plugins/modules/redfish_command.py

```diff
@@ -541,6 +547,26 @@
         username: operator
         password: supersecretpwd
 
+  - name: Multipart HTTP push update
+    community.general.redfish_command:
+      category: Update
+      command: MultipartHTTPPushUpdate
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
```

Comment:
Should I include "timeout" in the example here too? Depending on the size of the image and the transfer speed to the service, it might be needed. Is it worth trying to adjust the timeout without the user requesting it?

Reply:
@felixfontein curious what your thoughts are on this. Should there be warnings about this type of command taking a long time and advise users to adjust their timeout?

Reply:
It's probably a good idea to mention that in the documentation and/or examples.

Reply:
Okay, I'll push some doc changes that explain the need to provide `timeout`.

Reply:
Pushed doc updates for this.